src/parsers/transformer.rs

```diff
@@ -19,7 +19,7 @@ use uuid::Uuid;
 static UNIQUE_INTEGER: AtomicUsize = AtomicUsize::new(0);
 
 fn get_unique() -> usize {
-    return UNIQUE_INTEGER.fetch_add(1, Ordering::SeqCst);
+    UNIQUE_INTEGER.fetch_add(1, Ordering::SeqCst)
 }
 
 pub fn transform<'line>(value: &'line str, transformer: &Transformer, table_name: &str) -> String {
```

Comment:
Can line 34 below be changed to check the data_type instead of if it starts and ends with { and }

Reply:
100% at the mo in this PR im reading types, and not using them at all...

when i do the next bit i will change that